_ont/ont-odi-zte-dfp-34x-2c2.md

```diff
@@ -1,40 +1,9 @@
 ---
-title: ODI ZTE DFP-34X-C2C
```

Comment:
Isn't it better to delete the file?

Reply:
The ideal is to make the SEO realise that it has been moved and in a month's time delete it.